res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -614,12 +603,15 @@ LongShortDoubleBtn.prototype.ButtonDown = function(channel, control, value, stat
         this.ButtonCount = 1;
         // and short press
         this.ButtonLongPress = false;
-        this.ButtonLongPressTimer = engine.beginTimer(this.LongPressThreshold, function() {
-            myself.ButtonAssertLongPress();
-        }, true);
-        this.ButtonTimer = engine.beginTimer(this.DoublePressTimeOut, function() {
-            myself.ButtonAssert1Press();
-        }, true);
+
+        this.ButtonLongPressTimer = engine.beginTimer(
+            this.LongPressThreshold, this.ButtonAssertLongPress, true
```

Comment:
Oh, the ButtonAssertLongPress function refers to `this`, so maybe it wouldn't have worked. Anyway, looks good now.